README.md

```diff
@@ -11,16 +11,16 @@
 | :----------------------- | :----------------------------- |
 | [![REUSE status][9]][10] | [![ort-talk][11]][12]          |
 
-[1]: https://travis-ci.com/heremaps/oss-review-toolkit.svg?branch=master
-[2]: https://travis-ci.com/heremaps/oss-review-toolkit
+[1]: https://travis-ci.com/oss-review-toolkit/ort.svg?branch=master
+[2]: https://travis-ci.com/oss-review-toolkit/ort
 [3]: https://ci.appveyor.com/api/projects/status/hbc1mn5hpo9a4hcq/branch/master?svg=true
-[4]: https://ci.appveyor.com/project/heremaps/oss-review-toolkit/branch/master
-[5]: https://jitpack.io/v/heremaps/oss-review-toolkit.svg
-[6]: https://jitpack.io/#heremaps/oss-review-toolkit
-[7]: https://codecov.io/gh/heremaps/oss-review-toolkit/branch/master/graph/badge.svg
-[8]: https://codecov.io/gh/heremaps/oss-review-toolkit/
-[9]: https://api.reuse.software/badge/github.com/heremaps/oss-review-toolkit
-[10]: https://api.reuse.software/info/github.com/heremaps/oss-review-toolkit
+[4]: https://ci.appveyor.com/project/oss-review-toolkit/ort/branch/master
+[5]: https://jitpack.io/v/oss-review-toolkit/ort.svg
+[6]: https://jitpack.io/#oss-review-toolkit/ort
+[7]: https://codecov.io/gh/oss-review-toolkit/ort/branch/master/graph/badge.svg
+[8]: https://codecov.io/gh/oss-review-toolkit/ort/
+[9]: https://api.reuse.software/badge/github.com/oss-review-toolkit/ort
+[10]: https://api.reuse.software/info/github.com/oss-review-toolkit/ort
```

Comment:
Note to myself: I expect a breakage here as ORT had to be registered at the REUSE site, and we probably also need to update the URL there.

Reply:
Done, see https://api.reuse.software/info/github.com/oss-review-toolkit/ort.